push

```diff
@@ -94,7 +94,7 @@ def _find_utils():
     return ssh, scp, cmake
 
 def _restart_robot(host, ssh):
-    _ssh(ssh, host, 'systemctl restart opentrons-robot-server')
+    _ssh(ssh, host, 'systemctl restart opentrons-robot-server &')
```

Comment:
```suggestion
    _ssh(ssh, host, 'nohup systemctl restart opentrons-robot-server')
```

I think we want to fully nohup it rather than just background

Reply:
We can do both, so we free up the session right away.